firestore-bigquery-export/extension.yaml

```diff
@@ -352,3 +361,50 @@ params:
         value: no
     default: no
     required: true
+        
+  - param: DO_BACKFILL
+    label: Import existing Firestore documents into BigQuery
+    description: >-
+      Do you want to import existing documents from your Firestore collection into BigQuery? These documents 
+      will have each have a special changelog with the operation of `IMPORT` and the timestamp of epoch.
+      This ensures that any operation on an imported document supersedes the import record.
+    type: select
+    default: false
+    options:
+      - label: Yes
+        value: true
+      - label: No
+        value: false
+  
+  - param: IMPORT_COLLECTION_PATH
+    label: Existing documents collection
+    description:  >-
+      What is the path of the the Cloud Firestore Collection you would like to import from?
+      (This may, or may not, be the same Collection for which you plan to mirror changes.)
+    type: string
+    example: posts
+    validationRegex: "^[^/]+(/[^/]+/[^/]+)*$"
+    validationErrorMessage: Firestore collection paths must be an odd number of segments separated by slashes, e.g. "path/to/collection".
+    default: posts
+    required: true
+
+  - param: DOCS_PER_BACKFILL
+    label: Docs per backfill
+    description:  >-
+      When importing existing documents, how many should be imported at once?
+      The default value of 200 should be ok for most users.
+      If you are using a transform function or have very large documents, you may need to set this to a lower number.
+    type: string
+    example: 200
+    validationRegex: "^[1-9][0-9]*$"
+    validationErrorMessage: Must be a postive integer.
+    default: 200
+    required: true
+  
+lifecycleEvents:
+  onInstall:
+    function: fsimportexistingdocs
+    processingMessage: Checking whether to import existing documents
+  onUpdate:
+    function: fsimportexistingdocs
+    processingMessage: Checking whether to import existing documents
```

Comment:
Also onConfigure in case they change their mind about importing existing docs. This could be useful if the value for DOCS_PER_BACKFILL was too high and things failed, and then they want to re-configure it to be lower and try again...